db/migrate/20190118110059_add_service_offering_icons.rb

```diff
@@ -0,0 +1,16 @@
+class AddServiceOfferingIcons < ActiveRecord::Migration[5.2]
+  def change
+    create_table "service_offering_icons", :id => :serial, :force => :cascade do |t|
+      t.references :tenant, :type => :bigint, :index => true, :null => false, :foreign_key => {:on_delete => :cascade}
+      t.references :source, :type => :bigint, :index => false, :null => false, :foreign_key => {:on_delete => :cascade}
```

Comment:
Why not have an index on this one?

Reply:
yeah, adding specs for these. Source is already covered by the unique index

Reply:
Ah okay